lib/utils/amplitude.ts

```diff
@@ -1,19 +1,8 @@
 import amplitude from "amplitude-js"
+import { logAmplitudeEvent as dekoratorenLogEvent } from '@navikt/nav-dekoratoren-moduler';
 
 const isBrowser = () => typeof window !== "undefined"
 
-export const initAmplitude = () => {
-    if (isBrowser()) {
-        amplitude.getInstance().init("default", "", {
-            apiEndpoint: "amplitude.nav.no/collect-auto",
-            saveEvents: false,
-            includeUtm: true,
-            includeReferrer: true,
-            platform: window.location.toString(),
-        })
-    }
-}
-
 export function logAmplitudeEvent(
```

Comment:
Denne bør nok kalle `dekoratorenLogEvent` i stedet for amplitude-instansen som ble slettet over. Eks
```
export const logAmplitudeEvent = (eventName: string, eventData?: Record<string, any>) => {
  dekoratorenLogEvent({ origin: 'aap-kalkulator-frontend', eventName, eventData }).catch((e) => {
    console.log('Amplitude error', e);
  });
};
```